src/main/kotlin/com/terraformation/backend/search/table/ModulesTable.kt

```diff
@@ -0,0 +1,63 @@
+package com.terraformation.backend.search.table
+
+import com.terraformation.backend.auth.currentUser
+import com.terraformation.backend.db.accelerator.ModuleId
+import com.terraformation.backend.db.accelerator.tables.references.COHORT_MODULES
+import com.terraformation.backend.db.accelerator.tables.references.EVENTS
+import com.terraformation.backend.db.accelerator.tables.references.MODULES
+import com.terraformation.backend.db.accelerator.tables.references.PARTICIPANTS
+import com.terraformation.backend.db.default_schema.tables.references.PROJECTS
+import com.terraformation.backend.search.SearchTable
+import com.terraformation.backend.search.SublistField
+import com.terraformation.backend.search.field.SearchField
+import org.jooq.Condition
+import org.jooq.OrderField
+import org.jooq.Record
+import org.jooq.TableField
+import org.jooq.impl.DSL
+
+class ModulesTable(tables: SearchTables) : SearchTable() {
+  override val primaryKey: TableField<out Record, out Any?>
+    get() = MODULES.ID
+
+  override val sublists: List<SublistField> by lazy {
+    with(tables) {
+      listOf(
+          events.asMultiValueSublist("events", EVENTS.MODULE_ID.eq(MODULES.ID)),
+      )
+    }
+  }
+
+  override val fields: List<SearchField> =
+      listOf(
+          textField("additionalResources", MODULES.ADDITIONAL_RESOURCES, nullable = false),
```

Comment:
Aren't the text fields on this table all nullable except for `name`? The `nullable` flags here seem like they're inverted.